### PR TITLE
Additional information for the globemallow extension

### PR DESCRIPTION
**Pull request by mandyh101**
I have filled in the missing info for the Globemallow extension! Let me know if you think there is anything missing or to be improved! Thank you :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,37 @@
 # awesome-green-extensions
 A list of climate friendly extensions that can help you reduce the carbon footprint of your internet consumption. Work in progress. Needs links and descriptions and currently focused on Chrome.  Feel free to recommend more via Issue or ideally PR.
 
-## CO2 emissions trackers and green website raters
+# CO2 emissions trackers and green website raters
```
Comment (Leamsi9) [inline README.md]: This should be an H2 (##) because we are using H1(#) for the main title on line 1

File: README.md [hunk 2]
```diff
@@ -1,9 +1,37 @@
...
 
-### Globemallow
+## Globemallow
```
Comment (Leamsi9) [inline README.md]: This should be an h3

File: README.md [hunk 3]
```diff
@@ -1,9 +1,37 @@
...
+### Description
```
Comment (Leamsi9) [inline README.md]: And an h4 here (####)

**Review by Leamsi9 — CHANGES_REQUESTED** [review]
This is excellent @mandyh101 ! Better than I had designed it, really like the extra data you've added. Only a minor styling header hierarchy. Since the title on line 1 is an H1 (#) the remaining headers need to cascade accordingly, so need to add a hash to all the others. When that's done, we can merge it away! Thanks a lot!

**Comment by mandyh101** [comment]
Thanks @Leamsi9 I have updated the headings in the markdown - it looks much better now! I pushed these changes to my main branch so I believe the PR should be updated now too :)

**Pull request merged**